galley/pkg/config/analysis/analyzers/testdata/virtualservice_destinationhosts.yaml

```diff
@@ -60,3 +69,27 @@ spec:
     - destination:
         host: external-reviews.org  # Referring to a ServiceEntry host is valid and should not generate an error
                                     # Since this is an "external" service, subset is omitted
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: reviews-bookinfo-eu
+  namespace: default
+spec:
+  http:
+  - route:
+    - destination:
+        host: reviews.eu.bookinfo.com
```

Comment:
For this (and also the wildcard version) add a comment explaining the expected error / lack of error. See the other comments in this file for the pattern.